<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_CadToolBarDim</class>
 <widget class="QWidget" name="QG_CadToolBarDim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>65</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>65</width>
    <height>336</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dimensions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QToolButton" name="bBack">
     <property name="toolTip">
      <string>Back to main menu</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/back.png</normaloff>:/extui/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="bAligned">
     <property name="toolTip">
      <string>Aligned Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimaligned.png</normaloff>:/extui/dimaligned.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="bLinear">
     <property name="toolTip">
      <string>Linear Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimlinear.png</normaloff>:/extui/dimlinear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="bLinearHor">
     <property name="toolTip">
      <string>Horizontal Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimhor.png</normaloff>:/extui/dimhor.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="bLinearVer">
     <property name="toolTip">
      <string>Vertical Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimver.png</normaloff>:/extui/dimver.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="bRadial">
     <property name="toolTip">
      <string>Radial Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimradial.png</normaloff>:/extui/dimradial.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="bDiametric">
     <property name="toolTip">
      <string>Diametric Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimdiametric.png</normaloff>:/extui/dimdiametric.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="bAngular">
     <property name="toolTip">
      <string>Angular Dimension</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimangular.png</normaloff>:/extui/dimangular.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="bLeader">
     <property name="toolTip">
      <string>Leader</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/dimleader.png</normaloff>:/extui/dimleader.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bAligned</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimAligned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bLinearHor</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimLinearHor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bLinearVer</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimLinearVer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bLinear</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimLinear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBack</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bRadial</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimRadial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bDiametric</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimDiametric()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAngular</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimAngular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bLeader</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarDim</receiver>
   <slot>drawDimLeader()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
